package.json

```diff
@@ -27,7 +27,7 @@
     "startd": "nodemon src/",
     "debug": "nodemon --inspect=5959 src/",
     "debug:container": "nodemon --inspect=0.0.0.0:5959 src/",
-    "coverage": "cross-env NODE_ENV=test nyc --reporter=text-summary node_modules/.bin/_mocha --exit test/**/*.test.js",
+    "coverage": "cross-env NODE_ENV=test nyc --reporter=text-summary node_modules/.bin/_mocha --exit test/ --recursive",
```

Comment:
please revert; `/**/*.test.js` is what we need and is not equivalent to `--recursive`